content/docs/command-reference/params/index.md

```diff
@@ -45,9 +45,9 @@ parameters will be required for the stage invalidation (see `dvc status` and
 stage.
 
 Using parameter dependencies prevents situations where several
-[pipeline](/doc/command-reference/pipeline) stages depend on the same file, and
-any change in the file invalidates causes the reproduction all those stages
-unnecessarily.
+[pipeline](/doc/command-reference/pipeline) stages depend on the same
+configuration file, and any change in the file invalidates causes the
```

Comment:
I don't understand this change. What "configuration file"? I thought this sentence was about generic dependencies?

Maybe `...prevents situations where several pipeline stages share dependencies, and any change in those deps causes ..`